docs/app/src/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "root": true,
+  "extends": "../../../.eslintrc-browser.json",
+
+  "globals": {
+    // ngMocks
+    "module": false,
+    "inject": true,
```

Comment:
Good point; I'll add it to the list of things to be fixed once this PR lands. ;)